resources/pods.go

```diff
@@ -0,0 +1,39 @@
+package resources
```

Comment:
Was an accident. I moved it to a new directory and ended up moving it back and didn't pay attention to the other resource